packaging/hudi-spark-bundle/pom.xml

```diff
@@ -91,13 +91,15 @@
                   <include>org.jetbrains.kotlin:*</include>
                   <include>org.rocksdb:rocksdbjni</include>
                   <include>org.antlr:stringtemplate</include>
-                  <include>org.apache.parquet:parquet-avro</include>
 
+                  <include>com.lmax:disruptor</include>
```

Comment:
don't we want to shade this lib? it's purely internal right?

Reply:
I don't think we need to shade it, it's unlikely other engines will be using it

Reply:
let me add an open question in bundle standards, where we need to put out some guideline for shading